multichain/openrpc.yaml

```diff
@@ -5,179 +5,223 @@ info:
   description: >-
     This provides the specs for the MultiChain API Layer for the MetaMask API
 methods:
-  - name: "provider_authorize"
-    paramStructure: "by-name"
+  - name: provider_request
+    paramStructure: by-name
     params:
-      - name: "requiredScopes"
-        description: "Scopes that the wallet must support in order to be used with this provider."
+      - name: scope
+        description: a valid `scopeObject` that has been previously authorized via `provider_authorize`
+        required: true
+        schema:
+          $ref: "#/components/schemas/Scope"
+      - name: request
+        description: an object containing a JSON-RPC request with `method` and `params`
+        deprecated: false
+        required: true
         schema:
           type: "object"
+          properties:
+            method:
+              type: "string"
+            params:
+              true
+      - name: "sessionId"
+        description: >-
+          [CAIP-171](https://chainagnostic.org/CAIPs/caip-171) `SessionId` referencing a known, open session
+        schema:
+          type: "string"
+    result:
+      name: provider_requestResult
+      schema:
+        true
+    examples:
+      - name: provider_requestExample
+        params:
+          - name: scope
+            value: eip155:1
+          - name: request
+            value:
+              method: eth_getBalance
+              params: []
+          - name: sessionId
+            value: "0xdeadbeef"
```

Comment:
is it only `value`s that should remain strings?

Reply:
if i did it with this:
`0xdeadbeef` without the string wrapping then yaml would think its a hex number

Reply:
![image](https://github.com/MetaMask/api-specs/assets/364566/1f70d975-4ffa-44e7-9a94-503112bd6a03)